src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import './TodoForm.scss';
+import usersFromServer from '../../api/users';
+import { Todo, User } from '../../types/types';
+import { getUserByTodo } from '../../services/posts';
+
+type Props = {
+  onSubmit: (todo: Todo) => void;
+};
+
+export const TodoForm: React.FC<Props> = ({ onSubmit }) => {
+  const [title, setTitle] = useState('');
+  const [isTitleErrorShow, setIsTitleErrorShow] = useState(false);
+
+  const [userId, setUserId] = useState(0);
+  const [isUserErrorShow, setIsUserErrorShow] = useState(false);
+
+  function handleTitleChange(event: React.ChangeEvent<HTMLInputElement>) {
+    setTitle((event.target.value));
```

Comment:
For what double brackets?
```suggestion
    setTitle(event.target.value);
```